src/common/components/clusterConfiguration/NetworkConfiguration.tsx

```diff
@@ -131,10 +137,7 @@ const NetworkConfiguration: React.FC<NetworkConfigurationProps> = ({
 
       {children}
 
-      {!isUserManagedNetworking &&
-        !!clusterFeatureSupportLevels &&
-        clusterFeatureSupportLevels['CLUSTER_MANAGED_NETWORKING_WITH_VMS'] === 'unsupported' &&
-        vmsAlert}
+      {!isUserManagedNetworking && isClusterManagedNetworkingWithVMsUnsupported && { vmsAlert }}
```

Comment:
please remove the {} around vmsAlert it's what caused the networking page to crash